Cargo.toml

```diff
@@ -19,6 +19,7 @@ members = [
   "fuel-tests",
   "fuel-txpool",
   "xtask",
+  "version-compatibility",
```

Comment:
@Voxelot wrote in the slack:
> Im thinking we should test if client will still work against 0.14.1. Since it's a new field it'd be nice if an older version of the server just ignored the request, but we should double check that this won't suddenly break the SDK etc

So I added tests to verify version compatibility between `0.14.1` and `0.14.2`